pages/jobs.tsx

```diff
@@ -1,29 +1,39 @@
-// import Link from 'next/link'
-import { useState, useEffect } from 'react'
+import { useState } from 'react'
 import { GetStaticProps } from 'next'
 import { siteConfig } from '@/const/meta'
 import Content from 'components/Layout/Content'
-import { Section, Title, SubTitle, TitleSmall, LinkContainer, LoadingText } from 'const/styles/pages/content'
-import { ExternalLink } from '@/const/styles/global'
+import { Section, Title, SubTitle, TitleSmall, LinkContainer } from 'const/styles/pages/content'
+import { DropDown, ExternalLink } from '@/const/styles/global'
 import SVG from 'react-inlinesvg'
 
 async function getJobs() {
-  const { api, deptID } = siteConfig.greenhouse
-  const getJobs = await fetch(api)
-  const { jobs } = await getJobs.json()
-  const jobsData = []
-  await jobs.map((job) => {
-    const dept = job.departments[0]
-    if (dept) {
-      dept.id === deptID ? jobsData.push(job) : null
-    }
-  })
+  const jobsData = {}
+  const { api } = siteConfig.greenhouse
+
+  try {
+    const response = await fetch(api)
+    const data = await response.json()
+    data.jobs.map(job => {
```

Comment:
You are not returning anything from the `map`, so a better function for this is `forEach`

Reply:
Ah right, I'm not storing the return value of the `.map` in a variable. Updated and thanks for reviewing.